templates/hooks/group/sdk_update_post_build_request.go.tpl

```diff
@@ -0,0 +1,3 @@
+    if desired.ko.Spec.Path != nil {
+        input.NewPath = desired.ko.Spec.Path
+    }
```

Comment:
Here and for user resource, please add a code comment explaining why we need to do this. :)

Reply:
Addressed